docs/ref/models/database-functions.txt

```diff
@@ -762,6 +825,74 @@ Replaces all occurrences of ``text`` with ``replacement`` in ``expression``.
 The default replacement text is the empty string. The arguments to the function
 are case-sensitive.
 
+Usage example::
+
+    >>> from django.db.models import Value
+    >>> from django.db.models.functions import Replace
+    >>> Author.objects.create(name='Margaret Johnson')
+    >>> Author.objects.create(name='Margaret Smith')
+    >>> Author.objects.update(name=Replace('name', Value('Margaret'), Value('Margareth')))
+    2
+    >>> Author.objects.values('name')
+    <QuerySet [{'name': 'Margareth Johnson'}, {'name': 'Margareth Smith'}]>
+
+``Ord``
+-------
+
+.. class:: Ord(expression, **extra)
+
+.. versionadded:: 2.1
+
+Accepts a single text field or expression and returns the Unicode code point value
+for the first character of that expression.  It acts similar to Python's `ord`
+function, but the database backend will not raise an exception if the expression
+is more than one character long.
+
+It can also be registered as a transform as described in :class:`Length`.
+
+Usage example::
+
+    >>> from django.db.models.functions import Ord
+    >>> Author.objects.create(name='Margaret Smith')
+    >>> author = Author.objects.annotate(name_code_point=Ord('name')).get()
+    >>> print(author.name_code_point)
+    77
+
+``Right``
+---------
+
+.. class:: Right(expression, length, **extra)
+
+.. versionadded:: 2.1
+
+Accepts a single text field or expression and the 1-indexed length of the
+substring to return.  The first ``length`` characters are returned starting
+from the end of the string.  If ``length`` is negative, None is returned.
+
+Usage example::
+
+    >>> from django.db.models.functions import Right
+    >>> Author.objects.create(name='Margaret Smith')
+    >>> author = Author.objects.annotate(last_letter=Right('name', 1)).get()
+    >>> print(author.last_letter)
+    h
+
+..
+    Oracle and SQLite do not have a Right builtin function.  Django will
```

Comment:
I'm not sure if the implementation detail is important for an end user to care about?

Reply:
I tend to agree, @timgraham .  @atombrella suggested adding extra information about Oracle and SQLite.  Are you referring to the note about Oracle and SQLite?